sdks/python/apache_beam/metrics/cells_test.py

```diff
@@ -86,35 +86,42 @@ def test_parallel_access(self):
     for t in threads:
       t.join()
 
-    total = (self.NUM_ITERATIONS * (self.NUM_ITERATIONS-1)/2 * self.NUM_THREADS)
+    total = ((self.NUM_ITERATIONS + 1) *
+             (self.NUM_ITERATIONS)/2 *
+             self.NUM_THREADS)
+
+    count = ((self.NUM_ITERATIONS + 1)* self.NUM_THREADS)
 
-    count = (self.NUM_ITERATIONS * self.NUM_THREADS)
+    squares = (self.NUM_ITERATIONS *
```

Comment:
Add a comment about this? At first glance it seems unrelated to the squares, so it would be good to explain why this is related.